external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -277,7 +277,9 @@ private void emit() {
         waitingToEmit.remove();
     }
 
-    // emits one tuple per record
+    /**
+     * Emits one tuple per record
+     */
```

Comment:
This is a private method so it should be a standard line comment rather than a javadoc.